### PR TITLE
Added new unif target: GEPRC Taker F722 SE

### PR DESCRIPTION
**Pull request by Guidus93**
Added GEPRC TAKER F722SE config file

### CODE REVIEW

**Comment by nerdCopter** [comment]
* will hardware have all those gyros?
* i dont think serial settings are supposed to be pre-defined
* i dont think ESC_SENSOR feature is suppose to be pre-defined -- may need verification from other devs.

**Comment by nerdCopter** [comment]
thank you; please wait for other dev commentary.

**Review by sugaarK — DISMISSED** [review]
Given the plug fest going on I think it’s ok to define some serial connections..  @Guidus93 coild you send a couple of samples out to the guys? I’ll give you addresses on discord

File: configs/default/GEPR-TAKER_F722SE.config [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+#define USE_ACC
+#define USE_ACC_SPI_ICM42605
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO_SPI_MPU6000
+#define USE_ACCGYRO_BMI270
+#define USE_FLASH
+#define USE_FLASH_M25P16
+#define USE_FLASH_W25N01G
+#define USE_MAX7456
+#define USE_SDCARD
```
Comment (haslinghuis) [inline configs/default/GEPR-TAKER_F722SE.config:16]: USE_SDCARD needs to be removed.
```suggestion

```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

File: configs/default/GEPR-TAKER_F722SE.config [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42605
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC_SPI_MPU6000
+#define USE_ACC
+#define USE_ACC_SPI_ICM42605
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO_SPI_MPU6000
+#define USE_ACCGYRO_BMI270
```
Comment (haslinghuis) [inline configs/default/GEPR-TAKER_F722SE.config:11]: BMI[270] is only recommended for STM32F4xx

**Review by haslinghuis — CHANGES_REQUESTED** [review]
Just only see now you did not create feature branch. I will create a new one.

**Pull request closed without merging**